### DISCUSSION TITLE
Issues with padding and labels on RuxCheckbox in React

### DISCUSSION THREAD

**Discussion by aubsparrow** [RocketCommunicationsInc/astro#823] [Q&A] [answered]
I'm having an issue with RuxCheckbox in React where I have some text next to the checkbox and when I check the checkbox the padding seems to change and the text gets pushed over. 
![image](https://user-images.githubusercontent.com/50328837/195149071-a5f18171-8f12-4e28-94ed-efaf56b2f532.png)
![image](https://user-images.githubusercontent.com/50328837/195149158-5d3c5e1b-c635-493d-937e-238701396c9e.png)

Alternatively, if I change that text to be the label of the checkbox, when the box is checked the label disappears. 
![image](https://user-images.githubusercontent.com/50328837/195149453-94b9ff7d-7ba6-4b56-a04f-aec0423b901d.png)

**Comment by micahjones13** [comment, marked as answer]
Hey @aubsparrow, which version of `@astrouxds/react` are you using in this example?

**Reply by aubsparrow** [reply]
We are using 6.12.0

**Reply by micahjones13** [reply]
This issue was fixed in [version 6.13.1](https://github.com/RocketCommunicationsInc/astro/blob/main/packages/web-components/CHANGELOG.md#6131). There are no breaking changes in 6.13.1 from 6.12, so you shouldn't hit any snags when bumping the version. If you do though, please let us know!

**Reply by aubsparrow** [reply]
Did not have any issues. Thank you for your help!
